framework/sysdeps/linux/kvm.c

```diff
@@ -499,7 +499,7 @@ int kvm_generic_run(struct test *test, int cpu)
             ctx.vm_fd = kvm_generic_create_vm(kvm_fd);
             if (ctx.vm_fd < 0) {
                 if (errno == EBUSY) {
-                    log_info("SKIP reason: cannot create VM: device busy");
+                    log_skip(ResourceIssueSkipCategory, "SKIP reason: cannot create VM: device busy");
```

Comment:
Can you update this to remove the "SKIP reason:" portion, like #391 ?

Reply:
Done